data/autoyast_sle12sp2/http.sh

```diff
@@ -39,18 +39,18 @@ perl - opened <<EOP && IPTABLESRES=1
     \$proto = \$1;
     \$port = \$3;
     print "[INFO] port \$port/\$proto opened.\n";
-    #If tcp, expect 80/443/8080, for udp only 9090
-    if (\$proto eq 'tcp' && \$port =~ /80|443|8080/ || \$proto eq 'udp' && \$port eq 9090) {
+    #If tcp, expect 80/443/8080, for udp - 9090/5353
+    if (\$proto eq 'tcp' && \$port =~ /80|443|8080/ || \$proto eq 'udp' && \$port =~ /9090|5353/) {
         \$res += \$port;
         next;
     }
 
     print "[ERROR] Unexpected \$proto port \$port opened.\n";
     \$err=1;
   }
-  if (\$res != 17693 || \$err) {
+  if (\$res != 23046 || \$err) {
```

Comment:
We may get rid of this script, or move it in another repo, but I've removed magic numbers. See updated verification run.